src/explore-education-statistics-common/package.json

```diff
@@ -4,6 +4,7 @@
   "description": "",
   "devDependencies": {
     "@types/classnames": "^2.2.8",
+    "@types/element-resize-event": "^2.0.0",
```

Comment:
Remove this

Reply:
oops I was thinking about removing this so much I forgot to actually do it... doh